### ISSUE TITLE
inaccurate values with /v1/chain/get_account

### ISSUE THREAD

**Issue by AnSieger** [EOSIO/eos#8016]
I have an private eos network with an smart contract behaves like a table.
When an account interact with the contract and add an row, staked cpu from account should spent.
To observe the behavior i wrote an programm which log's every transaction.

Mostly the values are comprehensibly but sometimes the used cpu value shrinking (account cpu). The EOS Network running like 25 hours. So the staked cpu should not returned (returned after 72 hours). 

Is there an explanation?
Is the /v1/chain/get_account interface inaccurate ?

My log's:

account_name | head_block_num | head_block_time | cpu_limit_used | cpu_limit_available | diff_cpu
-- | -- | -- | -- | -- | --
user1 | 34393 | 2019-10-01T10:55:29.000 | 761553 | 987427809875 |  
user1 | 34394 | 2019-10-01T10:55:29.500 | 762027 | 987427809401 | 474
user1 | 34395 | 2019-10-01T10:55:30.000 | 761906 | 987427809522 | -121
user1 | 34398 | 2019-10-01T10:55:31.500 | 762151 | 987427809277 | 245
user1 | 34399 | 2019-10-01T10:55:32.000 | 762190 | 987427809238 | 39
user1 | 34401 | 2019-10-01T10:55:33.000 | 762554 | 987427808874 | 364
user1 | 34403 | 2019-10-01T10:55:34.000 | 762869 | 987427808559 | 315
user1 | 34404 | 2019-10-01T10:55:34.500 | 762912 | 987427808516 | 43
user1 | 34407 | 2019-10-01T10:55:36.000 | 763157 | 987427808271 | 245
user1 | 34408 | 2019-10-01T10:55:36.500 | 763521 | 987427807907 | 364
user1 | 34409 | 2019-10-01T10:55:37.000 | 763508 | 987427807920 | -13
user1 | 34411 | 2019-10-01T10:55:38.000 | 763833 | 987427807595 | 325

**Comment by jgiszczak** [comment]
CPU and network stake use begins decaying immediately after a transaction.  The 72 hours is approximately how long it takes to return all of the used stake.

Note that CPU and network stake used is only updated after a transaction.  `get_account` only retrieves the last calculated value.  This can result in apparently divergent results.  For accounts which don't produce a steady stream of transactions you may wish to duplicate the stake return equations and run them yourself, only periodically confirming with the chain.

**Comment by AnSieger** [comment]
Thank you for your feedback. I observed the behavior from transactions and resources again.
I found out that sometimes a transaction will be inexplicably canceled after 15-30 sec.
In this time the resource values "fluctuate".

For better understandig:
- I do the transaction (Contract do: insert value in table, EOS Token transfer)
- I can see the new value in table
- resource values "fluctuate" (15-30sec)
- Value in table disappear
- resource values "fluctuate" stopps. No resources were consumed. EOS Token are back.

Look like the transaction is not valid.
I did everything with one node. Next time i will use multipe nodes and observe the behavior
